ontology/ontoneo.md

```diff
@@ -2,32 +2,43 @@
 layout: ontology_detail
 id: ontoneo
 title: Obstetric and Neonatal Ontology
-browsers:
-- title: BioPortal Browser
-  label: BioPortal
-  url: https://bioportal.bioontology.org/ontologies/ONTONEO
-build:
-  source_url: http://purl.obolibrary.org/obo/ontoneo/ontoneo.owl
 contact:
   email: fernanda.farinelli@gmail.com
   github: FernandaFarinelli
   label: Fernanda Farinelli
   orcid: 0000-0003-2338-8872
 description: The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.
-domain: health
-homepage: ontoneo.com
 license:
-  label: CC BY 3.0
-  url: http://creativecommons.org/licenses/by/3.0/
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0
+domain: biomedical, health, obstetric and neonatal domain, gynecological domain, women's 
+homepage: ontoneo.com 
 mailing_list: http://groups.google.com/group/ontoneo-discuss
-preferredPrefix: ONTONEO
+publications: 
 products:
 - id: ontoneo.owl
+  title: ONTONEO
+  description: The full version of ONTONEO in OWL format 
+build:
+ source_url: https://raw.githubusercontent.com/ontoneo-project/Ontoneo/refs/heads/master/ontoneo.owl
+  - id: ontoneo.owl
+tracker: https://github.com/ontoneo-project/Ontoneo/issues
+browsers:
+  - label: BioPortal
+    title: BioPortal Browser
+    url: https://bioportal.bioontology.org/ontologies/ONTONEO
+  - label: OntoBee
+    title: OntoBee Browser
+    url: https://ontobee.org/ontology/ONTONEO
+publications:
+- id: https://ceur-ws.org/Vol-1747/IT403_ICBO2016.pdf
+  title: OntONeo: The Obstetric and Neonatal Ontology
 repository: https://github.com/ontoneo-project/Ontoneo
-tags:
-- biomedical
 tracker: https://github.com/ontoneo-project/Ontoneo/issues
+usages:
+- user: http://recol.eci.ufmg.br/
+  description: The Research Group on Knowledge Representation, Ontologies, and Language (ReCOL) investigates theoretical and applied aspects of knowledge modeling, including the use of biomedical ontologies such as OntONeo.
 activity_status: active
 ---
-
 The OntONeo suite is a collection of open ontologies available about Obstetric and Neonatal domain, currently designed to be composed by three complementary sub-ontologies covering several data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.
+---
```

Comment:
Remove since the text should be in the body of the markdown and not in the front matter.
```suggestion
```